content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -5,18 +5,38 @@ title: Using a KMS provider for data encryption
 content_type: task
 ---
 <!-- overview -->
-This page shows how to configure a Key Management Service (KMS) provider and plugin to enable secret data encryption.
+This page shows how to configure a Key Management Service (KMS) provider and plugin to enable secret data encryption. Currently there are two KMS API versions. KMS v1 will continue to work while v2 develops in maturity. If you are not sure which KMS API version to pick, choose v1. 
 
 ## {{% heading "prerequisites" %}}
 
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{< include "task-tutorial-prereqs.md" >}}
 
+The version of Kubernetes that you need depends on which KMS API version
+you have selected. 
+
+- If you selected KMS API v1, any supported Kubernetes version will work fine.
+- If you selected KMS API v2, you should use Kubernetes v{{ skew currentVersion }}
```

Comment:
```suggestion
- If you selected KMS API v2, you should use Kubernetes v{{< skew currentVersion >}}
```
nit: looked at the rendered docs and this isn't formatted: https://deploy-preview-35046--kubernetes-io-vnext-staging.netlify.app/docs/tasks/administer-cluster/kms-provider/